react-cors-spa/package.json

```diff
@@ -5,19 +5,22 @@
   "email": "jguilloi@amazon.fr",
   "license": "MIT",
   "devDependencies": {
+    "@tanstack/eslint-plugin-query": "^5.53.0",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "react-scripts": "^5.0.1"
   },
   "dependencies": {
-    "axios": "^1.7.5",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "@tanstack/react-query": "^5.54.1",
+    "@tanstack/react-query-devtools": "^5.54.1",
+    "axios": "^1.7.7",
+    "react": "19.0.0-rc-4c58fce7-20240904",
+    "react-dom": "19.0.0-rc-4c58fce7-20240904",
     "react-router-dom": "^5.3.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "NODE_ENV=development & react-scripts start",
```

Comment:
Same issue as below

Reply:
I'll look into checking the URL instead - I like the sound of that option as well